SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -938,6 +938,21 @@ void CNEMOEulerSolver::Source_Residual(CGeometry *geometry, CSolver **solver_con
         /*--- If necessary, set variables needed for viscous computation ---*/
         if (viscous) {
 
+          // TODO: NOTE: Some of these are set above. They need to be set here as well, otherwise they are passed in incorrectly.
+          // Do not remove.
+          
```

Comment:
This is not something that just happens.
Most likely scenario is that you have a memory leak, which should be easy to find with valgrind.
Compile a debug or debugoptimized version, choose a small testcase that uses this feature, run
valgrind SU2_CFD your_config.cfg
see where it leads you.

Reply:
Any luck @jtneedels ? We were planning to release 7.1.1 but we can also wait a little more.

Reply:
@pcarruscag I haven't resolved it yet, but I am working on it more today. What is the timeline for the version 7.1.1 release? I don't want to hold it up.

Reply:
We don't have strict deadlines but we try to have these maintenance releases every month.
This month is shorter, so if you want a specific day lets say Monday.

Reply:
Okay sounds good. I'm definitely seeing a memory leak, and a number of uninitialized warnings in CNEMOEulerVariable.cpp and CNEMOEulerSolver.cpp. Still working on finding what's causing this specific issue.

Reply:
Hi Pedro, I'm having trouble pinning down the memory leak causing the issue, but it looks like it might be coming from the Auxvar_grad terms used in the source residual. I'm having trouble getting specific line numbers when running valgrind with: valgrind --leak-check=full --track-origins=yes SU2_CFD testcase.cfg and compiling with debugoptimized. Is there anything I need to add to my run command?

Reply:
debug instead of debug optimized might give more information... I do not know of anything else you may add.
Some of those warnings are infectious in that if something is not initialized all dependent quantities will also be a warning.
You need to start fixing from the top down. And then because of how the numerics (in general not just NEMO) are implemented the bug might be due to something else...